buildozer/README.md

```diff
@@ -0,0 +1,106 @@
+# Buildozer
+
+Buildozer is a command line tool to refactor multiple [Bazel](https://github.com/bazelbuild/bazel) BUILD files using standard commands. 
```

Comment:
I think 'rewrite' is more accurate than' refactor'
Also, line break after 80 cols